README.md

```diff
@@ -10,18 +10,21 @@ Install with [npm](http://npmjs.org/)
 npm install indico.io
 ```
 
-Documentation
-------------
-Found [here](http://indico.readme.io/v1.0/docs)
-
-### API
+API Keys + Setup
+----------------
+For API key registration and setup, checkout our [quickstart guide](http://docs.indico.io/v2.0/docs/api-keys).
 
-Right now this wrapper supports the following apps:
+Full Documentation
+------------
+Detailed documentation and further code examples are available at [indico.reame.io](http://indico.readme.io/v2.0/docs/node).
 
-- Political Sentiment Analysis
+### Supported APIs
+- Sentiment Analysis
+- Text Tagging
+- Political Analysis
 - Language Detection
-- Positive/Negative Sentiment Analysis
 - Facial Emotion Recognition
+- Image Feature Extraction
 - Facial Feature Extraction
 
 ### Using the library
```

Comment:
We call this header 'Examples' in our other readmes. Probably worth being consistent.